src/process/process.rs

```diff
@@ -63,8 +63,9 @@ pub struct Callback {
     pub pc: usize
 }
 
-#[repr(C,packed)]
```

Comment:
The (Rustnomicon)[https://doc.rust-lang.org/nomicon/other-reprs.html] heavily discourages you from using `packed`. Basically, there's no guarantee that the platform will actually be able to work with a packed struct if it doesn't support unaligned accesses, and `repr(C)` already gives you enough guarantees that no extraneous padding will be introduced. I think if we wanted to be extra safe we could switch to using `u32` here instead, but as is we don't even make sure to deal with endianess when writing the struct so I think it's a moot point. Does this sound reasonable?